tasks/rh-sign-cosign-image/README.md

```diff
@@ -0,0 +1,16 @@
+# rh-sign-cosign-image
+
+Task to create sign containers in snapshot with cosign
```

Comment:
I think the word create in this line was accidentally leftover

Reply:
Fixed in https://github.com/konflux-ci/release-service-catalog/pull/467.

Reply:
Thanks.